### ISSUE TITLE
Add more mythical creatures the originate from Asia

### ISSUE THREAD

**Issue by Spinoyonyx** [AlexModGuy/Ice-and-Fire-Suggestions#782]
When i see the whole roster of creatures and planned creatures so far i see there is no asian mythical creatures at all in the whole roster of mobs so i got some ideas

Add creatures like the a Chinese or Japanese dragon(a Ryu or Long/Lung/龙) which maybe thhey can be tamed by offering it gold since its origin is from being more praised then getting slayed and maybe make them Neutral and make them spawn in the Ocean biome or in a Pagoda temple.

 Maybe add a Phaya Naga that spawn in rivers(Since it is believed to live in the Mekong River) and make it an untameable River monster

U can also add Chinese Guardian Lion that are first statues then awaken once u stole a treasure they guard and maybe u can tame them in a way and become a guardian pet that attacks when the player is attacked and awaken. After killing the mob they return back to the home they set and become a statue again.

Maybe even add the other half animal half dragon type creatures from Chinese folklore like the Qilin/Kirin a Chinese/Japanese unicorn or the Lóngguī(Dragon turtle

**Comment by PTZ8** [comment]
Yea that’s what I was thinking too!

**Comment by PTZ8** [comment]
Sounds really good.

**Comment by HeroNeeb** [comment]
Yeah that would be a good Idea but... It's a Greek Mythology Mod maybe if they would do antoher Asian Mod they could add those Creatures. But it's a nice Idea :-)

**Comment by Spinoyonyx** [comment]
> Yeah that would be a good Idea but... It's a Greek Mythology Mod maybe if they would do antoher Asian Mod they could add those Creatures. But it's a nice Idea :-)

isnt a fire  and ice dragon, werewolf,wendigo,and vampire not from greek mythology?

**Comment by Comdec35000** [comment]
We can also imagin Onis (鬼) and Kitsune (狐) from japanese mythology. Onis may looks like realy big and fat troll and kitsune may have more tail along they grows... 
Why not adding mask like Oni mask and fox mask like theses : 
![kitsune](https://user-images.githubusercontent.com/66130341/84694408-70e8cd80-af49-11ea-95f4-d8335385cb78.jpg)

![71C9ZaolLvL _AC_SL1150_](https://user-images.githubusercontent.com/66130341/84694470-89f17e80-af49-11ea-9d6b-8fd7532ffd67.jpg)
 witch add some custom effect

there sould be a lot of type of kitsune/oni mask, as cosmethics

**Comment by TheDerpiB0ss** [comment]
oni mask can have a type of effect like if you found one in nether that he will have fire resistance, or you found one in deep ocean who can have water breathing

**Comment by PTZ8** [comment]
These types are very nice
